scripts/import_dataset.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -e
+# Copyright 2020. IBM All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BASEDIR="$(dirname "$0")"
+source ${BASEDIR}/env.sh
+
+# Install aria and get example dataset zip file
+echo "Installing aria2..."
+apt-get -o Dpkg::Use-Pty=0 update -qq  || echo " RC${?} Got an error on update???"
+apt-get -o Dpkg::Use-Pty=0 install -qq aria2
+
+cd $HOME
+echo "Fetching example dataset from ${URLPAIVDATASET}"
+aria2c -q $URLPAIVDATASET
```

Comment:
switch to a simple wget or curl (with appropriate flags) to output the file where you want it.